tests/e2e/helpers.ts

```diff
@@ -172,6 +187,15 @@ export const WALLETS_WITH_COINS = {
   'Standard HD Segwit': {
     SEED_PHRASE: 'patrol correct rural firm special eight smooth orphan pilot oppose wealth rival',
   },
+  // Wallet for receiving coins from send and cancel tests.
+  '3-Key moneybox': {
```

Comment:
Why can't use some of the wallets above? And also, please notice this object has a completely different structure. IMHO you should add missing fields or extract it to a variable

Reply:
I only aligned the structure